apps/nsq_stat/nsq_stat.go

```diff
@@ -120,6 +122,14 @@ func main() {
 		log.Fatal("--topic and --channel are required")
 	}
 
+	if int64(*statusEvery) <= 0 {
+		log.Fatal("--status-every should be positive")
+	}
+
+	if int64(*countNum) < 0 {
+		log.Fatal("--count should be positive or equals zero")
```

Comment:
Yes, it sounds logical. But `-count` is optional flag and must to have default value because of flag API.
So I don't know how to solve it in elegant way.